infra/test/setup/iam.tf

```diff
@@ -16,7 +16,19 @@
 
 locals {
   int_required_roles = [
-    "roles/owner"
+    "roles/browser",
```

Comment:
suggestion: alphabetize the roles for easier review & comparison.

Reply:
sorted